frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tsx

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import { find, lowerCase, noop } from "lodash";
+import { formatDistanceToNowStrict } from "date-fns";
+
+import { ActivityType, IActivity } from "interfaces/activity";
+import { addGravatarUrlToResource } from "utilities/helpers";
+import Avatar from "components/Avatar";
+import Button from "components/buttons/Button";
+import Icon from "components/Icon";
+
+const baseClass = "activity-item";
+
+const DEFAULT_GRAVATAR_URL =
+  "https://www.gravatar.com/avatar/00000000000000000000000000000000?d=blank&size=200";
```

Comment:
I know you just copied this over, but it looks like this links to a blank image. Is that what we want to display? It looks like, for our use case, it may never get called because we always have an email for activity type. 

It looks like if there is no gravatar for an email, it defaults to https://i2.wp.com/fleetdm.com/images/permanent/icon-avatar-default-128x128-2x.png?ssl=1. Should we use the same URL here?

Reply:
yah we probably actually can convert it over to SVG. I'll look into changing that.